message_passing/update_with_start/lazy_initialization/__init__.py

```diff
@@ -0,0 +1 @@
+TASK_QUEUE = "uws"
```

Comment:
I think the task queue should be named specific to the sample so it doesn't clash with others (also, this constant isn't used in `starter.py` as might be expected)

Reply:
done